src/app/infobox/infobox.component.html

```diff
@@ -1,12 +1,32 @@
 <div *ngIf="this.description" class="card" [hidden]="!isVisible" [style.color]="themeService.cardColor">
-    <div>
-      <h2><b [style.color]="themeService.cardColor">{{this.title}}</b></h2><br><p class="query_description">{{this.querydescription}}</p><hr width="50%" align="left" *ngIf="this.image"><hr *ngIf="!this.image"><div class="info-img"><img class="main_image" *ngIf="this.image"  src={{this.image}}></div>
-      <p [style.color]="themeService.descriptionColor">
-      {{this.description}}<a [style.color]="themeService.linkColor" href='https://en.wikipedia.org/wiki/{{this.title}}'><br>More at Wikipedia</a>
-      <br>
-      <a href='https://twitter.com/search?src=typd&q={{this.title}}'><br>{{this.title}} on Twitter</a>
-      <br>
-      <a href='https://www.youtube.com/results?search_query={{this.title}}'><br>{{this.title}} on YouTube</a>   
-      </p>
+  <div>
+    <h2><b [style.color]="themeService.cardColor">{{this.title}}</b><i data-toggle="modal" data-target="#myModal" class="material-icons">share</i></h2><br><p class="query_description">{{this.querydescription}}</p><hr width="50%" align="left" *ngIf="this.image"><hr *ngIf="!this.image"><div class="info-img"><img class="main_image" *ngIf="this.image"  src={{this.image}}></div>
+    <p [style.color]="themeService.descriptionColor">
+    {{this.description}}<a [style.color]="themeService.linkColor" href='https://en.wikipedia.org/wiki/{{this.title}}'><br>More at Wikipedia</a>
+    <br>
+    <a href='https://twitter.com/search?src=typd&q={{this.title}}'><br>{{this.title}} on Twitter</a>
+    <br>
+    <a href='https://www.youtube.com/results?search_query={{this.title}}'><br>{{this.title}} on YouTube</a>   
+    </p>
+  </div>
+  <!-- Modal -->
+  <div class="modal fade" id="myModal" role="dialog">
```

Comment:
If this is not material modal provided by angular then I would suggest using that instead of your own modal.

**myModal** is not a good name for an ID.